packages/docs-site/blog/2023/06/29/new-language-features.md

```diff
@@ -0,0 +1,302 @@
+<script setup>
+import BlogMeta from "../../../../src/components/BlogMeta.vue";
+</script>
+
+# What Have We Done to the Languages?
+
+<BlogMeta github="liangyiliang" date="2023-06-13" />
+
+I joined the Penrose team in July 2022, and worked on of a few language-related improvements to Penrose which have the potential of making Penrose more flexible.
+
+## Binary Symmetric Predicates
+
+In July 2022, we implemented support for binary symmetric predicates which allowed for more flexible style-substance matching. By marking a binary predicate with the `symmetric` keyword, the Domain author basically tells Penrose that it does not care about the order of input to the predicate, just like a symmetric mathematical relation.
```

Comment:
Might wanna open with: 

> Mathematical relations can be symmetric. For example, [something like +]. Penrose didn't have the notion of symmetry and one would need to duplicate code to simulate this. 

Personally, I find the article to be a bit easier to follow if it had started with the problem first.

Reply:
Done